app/controllers/api/v1/customers/random_customers_controller.rb

```diff
@@ -0,0 +1,5 @@
+class Api::V1::Customers::RandomCustomersController < ApplicationController
+  def show
+    render json: Customer.order("RANDOM()").first
```

Comment:
let's pull the logic needed to find the random customer out into a method in your `Customer` model. your controller doesn't need to know about this logic 👍